marlowe-dashboard-client/src/Play/View.purs

```diff
@@ -0,0 +1,206 @@
+module Play.View (renderPlayState) where
+
+import Prelude hiding (div)
+import Contract.Types (State) as Contract
+import Contract.View (contractsScreen, contractDetailsCard)
+import Css (classNames, hideWhen)
+import Data.Either (Either(..))
+import Data.Foldable (foldMap)
+import Data.Lens (view)
+import Data.Maybe (Maybe(..), isNothing)
+import Halogen.HTML (HTML, a, div, footer, h1, header, main, nav, span, text)
+import Halogen.HTML.Events (onClick)
+import Halogen.HTML.Properties (href)
+import MainFrame.Lenses (_card, _screen)
+import Marlowe.Semantics (PubKey)
+import Material.Icons as Icon
+import Play.Lenses (_contractState, _menuOpen, _templateState, _wallet)
+import Play.Types (Action(..), Card(..), ContractStatus(..), Screen(..), State)
+import Prim.TypeError (class Warn, Text)
+import Template.Types (Template)
+import Template.Types (State) as Template
+import Template.View (contractSetupConfirmationCard, contractSetupScreen, templateLibraryCard)
+import WalletData.Types (WalletLibrary, WalletNicknameKey)
+import WalletData.View (newWalletCard, walletDetailsCard, putdownWalletCard, walletLibraryScreen)
+
+renderPlayState :: forall p. WalletLibrary -> WalletNicknameKey -> Array Template -> State -> HTML p Action
+renderPlayState wallets newWalletNicknameKey templates playState =
+  let
+    wallet = view _wallet playState
+
+    menuOpen = view _menuOpen playState
+
+    screen = view _screen playState
+
+    card = view _card playState
+
+    templateState = view _templateState playState
+
+    contractState = view _contractState playState
+  in
+    div
+      [ classNames [ "grid", "h-full", "grid-rows-main" ] ]
+      [ renderHeader wallet menuOpen
+      , renderMain newWalletNicknameKey wallets templates wallet menuOpen screen card templateState contractState
+      , renderFooter
+      ]
+
+------------------------------------------------------------
+renderHeader :: forall p. PubKey -> Boolean -> HTML p Action
+renderHeader wallet menuOpen =
+  header
+    [ classNames [ "relative", "flex", "justify-between", "text-green" ] ]
+    [ h1
+        [ classNames $ itemClasses <> [ "cursor-pointer" ]
+        , onClick $ const $ Just $ SetScreen $ ContractsScreen Running
```

Comment:
seriously we have to think of a good name for this function and make it `onClick <<< const <<< Just`

Reply:
maybe `onClick_` like how we have `div_` which means the first parameter is empty, this is kind of the same, the function parameter is empty.